wp-content/themes/core/assets/pcss/color/_variables.pcss

```diff
@@ -13,69 +13,65 @@
 	 * Color Naming Site: https://chir.ag/projects/name-that-color/
 	 * ----------------------------------------------------------------------- */
 
-	--color-royal-blue: var(--wp--preset--color--royal-blue);
-	--color-jacksons-purple: var(--wp--preset--color--jacksons-purple);
+	--color-blue: var(--wp--preset--color--blue, #3050e5);
+	--color-blue-rgb: 48, 80, 229;
+	--color-blue-alpha-20: rgba(var(--color-blue-rgb), 0.2);
+	--color-dark-blue: #2238a0;
 
 	/* -----------------------------------------------------------------------
 	 * Neutral: Grey
 	 * ----------------------------------------------------------------------- */
 
-	--color-neutral-10: var(--wp--preset--color--neutral-10);
-	--color-neutral-20: var(--wp--preset--color--neutral-20);
-	--color-neutral-30: var(--wp--preset--color--neutral-30);
-	--color-neutral-40: var(--wp--preset--color--neutral-40);
-	--color-neutral-50: var(--wp--preset--color--neutral-50);
-	--color-neutral-60: var(--wp--preset--color--neutral-60);
-	--color-neutral-70: var(--wp--preset--color--neutral-70);
-	--color-neutral-80: var(--wp--preset--color--neutral-80);
-	--color-neutral-90: var(--wp--preset--color--neutral-90);
+	--color-neutral-10: var(--wp--preset--color--neutral-10, #f5f5f5);
+	--color-neutral-20: #e8e8e8;
+	--color-neutral-30: #d1d1d1;
+	--color-neutral-40: #a3a3a3;
+	--color-neutral-50: var(--wp--preset--color--neutral-50, #767676);
+	--color-neutral-60: #5d5d5d;
+	--color-neutral-70: #464646;
+	--color-neutral-80: #2f2f2f;
+	--color-neutral-90: #1f1f1f;
```

Comment:
Curious about this change. IIRC, our goal was to always define all properties via theme.json and then reference those values in our CSS to prevent mismatched values.

Reply:
The issue I've been running into on projects lately is that design wants all the colors available, but they don't want all the colors available to editors. So, we end up only adding a few of the colors to the global color palette and then the rest get added in code to be used elsewhere on the site. Normally these other grays are used in places not editable by editors like buttons and navigation. It's likely though that we're not using all of these colors and a lot of them can be stripped though. 

Design also wanted to go back to `primary` / `secondary` with this update - so maybe we have a pretty big disconnect on colors that we need to meet to go over? Maybe I can post in the channel first for feelers on if we need to meet or not?

There's always the option to get super granular with colors too, where we can choose per block which colors are available in the palette.